contracts/Core/StakeManager.sol

```diff
@@ -116,15 +109,10 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         stakers[stakerId].epochLastCommitted = parameters.getEpoch();
     }
 
-    function updateBlockReward(uint256 _blockReward) external onlyRole(parameters.getDefaultAdminHash())
-    {
-        blockReward = _blockReward;
-    }
-
     /// @notice stake during commit state only
     /// we check epoch during every transaction to avoid withholding and rebroadcasting attacks
     /// @param epoch The Epoch value for which staker is requesting to stake
-    /// @param amount The amount of schelling tokens Staker stakes
+    /// @param amount The amount in RZR
```

Comment:
Ideally this must be renamed in a separate PR to address #156. Also, RAZOR*

Reply:
Sure will make this change

Reply:
makes sense. Lets formalise it while addressing #156.